sdk/python/examples/descriptors/nem_mosaic.py

```diff
@@ -0,0 +1,87 @@
+from symbolchain.core.facade.NemFacade import Address
+from symbolchain.nc import Address as sc_Address
+from symbolchain.nc import Amount, MosaicId, MosaicLevy, MosaicTransferFeeType, NamespaceId
+
+
+def descriptor_factory():
+	sample_address = 'TALICEROONSJCPHC63F52V6FY3SDMSVAEUGHMB7C'
+
+	# HACK: until fixed
+	levy = MosaicLevy()
+	levy.transfer_fee_type = MosaicTransferFeeType.ABSOLUTE
+	levy.recipient_address = sc_Address(str(Address(sample_address)).encode('utf8'))
+	levy.mosaic_id = MosaicId()
+	levy.mosaic_id.namespace_id = NamespaceId()
+	levy.mosaic_id.namespace_id.name = 'lieutenant'.encode('utf8')
+	levy.mosaic_id.name = 'colonel'.encode('utf8')
+	levy.fee = Amount(632_0000)
+
```

Comment:
I'm not sure, but I _think_ fix would be `sizeof`, like we have for `SizePrefixed*` things?
```
	modification_size = sizeof(uint32, modification)

	modification = MultisigAccountModification
```

so something like

```
	levy_size = sizeof(uint32, levy)

	levy = MosaicLevy if 0x00000000 not equals levy_size
```

although not super sure about it, also it looks potentially problematic:
 1. in case of current generator: doing `sizeof`, would skip the field from being present, i.e.
 ```
 	def serialize(self) -> bytes:
		buffer_ = bytes()
		buffer_ += self.mosaic_id.size().to_bytes(4, byteorder="little", signed=False)  # mosaic_id_size
		buffer_ += self._mosaic_id.serialize()
		buffer_ += self._amount.serialize()
		return buffer_
```
(mosaic_id_size has size derived from mosaic_id itself, so it's not present as a separate field...)

 2. but at the same time condition is based on "levy_size" field...

Reply:
i guess we can try sizeof and see what happens